docs/pyboard/tutorial/amp_skin.rst

```diff
@@ -60,13 +60,37 @@ on your pyboard (either on the flash or the SD card in the top-level directory).
 or to convert any file you have with the command::
 
     avconv -i original.wav -ar 22050 -codec pcm_u8 test.wav
-    
-Then you can do::
+
+Or by using online converters like http://audio.online-convert.com/convert-to-wav
```

Comment:
-1. Sites like this come and go, no need to collect broken links in the docs. (Feel free to argue on this one.)